lib/utils.bash

```diff
@@ -212,7 +212,7 @@ install_version() {
       echo "* Configuring ffmpeg with default options"
     fi
 
-    ./configure --prefix="${install_path%/bin}" ${enabled_options} || exit 1
+    ./configure --prefix="${install_path%/bin}" ${enabled_options} ${ASDF_FFMPEG_OPTIONS_EXTRA} || exit 1
```

Comment:
Thanks, this is a good idea. I think this will fix the CI issue:

```suggestion
    ./configure --prefix="${install_path%/bin}" ${enabled_options} ${ASDF_FFMPEG_OPTIONS_EXTRA:-} || exit 1
```

Reply:
ok